generators/workspace/templates/package.tmpl.json

```diff
@@ -34,34 +34,34 @@
   },
   "devDependencies": {
     "clean-webpack-plugin": "~3.0.0",
-    "copy-webpack-plugin": "~6.0.3",
-    "css-loader": "~3.6.0",
+    "copy-webpack-plugin": "~6.3.2",
+    "css-loader": "~5.0.1",
     "css-minimizer-webpack-plugin": "^1.1.5",
-    "expose-loader": "~1.0.0",
-    "file-loader": "~6.0.0",
-    "html-loader": "~1.1.0",
-    "html-webpack-plugin": "~4.3.0",
+    "expose-loader": "~1.0.3",
+    "file-loader": "~6.2.0",
+    "html-loader": "~1.3.2",
+    "html-webpack-plugin": "~4.5.0",
     "identity-obj-proxy": "~3.0.0",
     "ifdef-loader": "~2.1.5",
-    "imports-loader": "~1.1.0",
-    "mini-css-extract-plugin": "~0.9.0",
-    "node-sass": "~4.14.1",
+    "imports-loader": "~1.2.0",
+    "mini-css-extract-plugin": "~1.3.2",
+    "node-sass": "~5.0.0",
     "npm-run-all": "~4.1.5",
-    "npm-watch": "^0.6.0",
-    "raw-loader": "~4.0.1",
+    "npm-watch": "^0.7.0",
+    "raw-loader": "~4.0.2",
     "rimraf": "~3.0.2",
-    "sass-loader": "~9.0.2",
-    "shx": "~0.3.2",
-    "source-map-loader": "~1.0.1",
-    "style-loader": "~1.2.1",
-    "tslib": "~2.0.0",
+    "sass-loader": "~10.1.0",
+    "shx": "~0.3.3",
+    "source-map-loader": "~1.1.3",
+    "style-loader": "~2.0.0",
+    "tslib": "~2.0.3",
     "tslint": "~5.20.1",
     "typedoc": "~0.17.8",
     "typescript": "~3.9.7",
-    "url-loader": "~4.1.0",
-    "webpack": "~4.43.0",
-    "webpack-bundle-analyzer": "~3.8.0",
-    "webpack-cli": "~3.3.12",
-    "webpack-dev-server": "~3.10.3"
+    "url-loader": "~4.1.1",
+    "webpack": "~4.44.2",
+    "webpack-bundle-analyzer": "~3.9.1",
+    "webpack-cli": "~4.2.0",
```

Comment:
```suggestion
    "webpack-cli": "~3.3.12",
```

This change causes the following error:

```
> tsc -w

[12:19:18 PM] Starting compilation in watch mode...

[12:19:18 PM] Starting compilation in watch mode...

internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module 'webpack-cli/bin/config-yargs'
Require stack:
- /home/holger/w/ordino_develop/node_modules/webpack-dev-server/bin/webpack-dev-server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/holger/w/ordino_develop/node_modules/webpack-dev-server/bin/webpack-dev-server.js:65:1)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/holger/w/ordino_develop/node_modules/webpack-dev-server/bin/webpack-dev-server.js'
  ]
}
```

The last version of the 3.x branch is `3.3.12`.